### DISCUSSION TITLE
Will VSA-DSS style model be implemented in QGEP?

### DISCUSSION THREAD

**Discussion by decheu** [QGEP/QGEP#723] [Ideas]
The VSA is preparing a unified style model (Darstellungsmodell) for Switzerland. The [canton of Zurich plans to use it](https://www.zh.ch/content/dam/zhweb/bilder-dokumente/themen/umwelt-tiere/wasser-gewaesser/gewaesserschutz/abwasserentsorgung/kgdm_129_gep_modelldokumentation.pdf).
Are we planning to implement it into QGEP?
Does anyone know when it will be published?
I guess that once the styling files published (hopefully something useful for QGIS as e. g. .qml), it should be possible to add them to QGEP with reasonable effort.

**Comment by sjib** [comment]
@decheu The cantons BE / SO have a draft for a Darstellungsmodel resp. Publikationsmodell: 
Im Rahmen der Implementation Informationsplattform Wasser IPW der Kantone Bern / Solothurn wurde ein Publikationsmodell für die definierten Pläne entwickelt, welches die Daten aus VSA-DSS-Mini 2020 in ein Publikationsmodell umwandelt. Die Dokumentation dazu findet sich online auf github:
https://github.com/ipw-informationsplattform/dssminipub_public

It bases on: 
https://www.bvd.be.ch/content/dam/bvd/dokumente/de/awa/wasser/abwasserentsorgung/genereller-entw%C3%A4sserungsplan-gep/wegleitung-daten-se-darstellungsmodell-gep-beilage5.pdf
and defines the following maps:

- Erschliessungsplan GEP (nur relevant für Kanton Solothurn)
-  GEP-Massnahmenplan (TP12)
- Plan Werkkataster IST-Zustand (TP04)
- Zustandsplan Kanalisation (TP05)
- Plan Abwasserbehandlung ländlicher Raum (ALR) (TP10)
- Zustandsplan Teileinzugsgebiet Ist (TP 11)
- Zustandsplan Teileinzugsgebiet Soll (TP 11)
- Konfliktplan Schutzzonen (TP12)

But this cannot be directly implemented on QGIS, as it bases on VSA-DSS-Mini but could be used as an inspiration.

The raumdatenpool Lucerne has published drawing rule for 
- Werkinformation (TP04)
- Datenverwaltung (TP03)
![grafik](https://github.com/QGEP/QGEP/assets/1265865/db8f5eef-17f5-4ff3-91ee-14dfb7eb2fbd)
see https://raumdatenpool.ch/Dokumente/RDP_Spezifikation_Datenbewirtschaftung_WI_Abwasser_GEP_v2.4.zip
and intends to go on with further Teilprojects (TP)
